From ad1afe394dba98be939ec3285efcf9ff87cbf246 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Tue, 25 May 2021 22:15:34 +0200
Subject: [PATCH] Fix DBus check

Android and macOS are Unix too, but we don't want to require DBus there
---
 CMakeLists.txt | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59a9a0e..c5b161f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,10 +15,12 @@ set(REQUIRED_QT_VERSION 5.15.0)
 
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Gui Sql Widgets Qml)
 
-# On UNIX DBus integration is required, on Android it does not exist and on other platforms it is optional
-if(UNIX)
-    find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED DBus)
-elseif(NOT ANDROID)
+# On non-Apple UNIX DBus integration is required, on Android it does not exist and on other platforms it is optional
+if(UNIX AND NOT APPLE)
+    if(NOT ANDROID)
+        find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED DBus)
+    endif()
+else()
     find_package(Qt5 ${REQUIRED_QT_VERSION} OPTIONAL_COMPONENTS DBus)
 endif()
 
-- 
2.31.1

